tests/tests/control_wam

```diff
@@ -1,46 +1,111 @@
 ###############################################################################
 #
-#  Global control multigases test
+#  Global fv3 wam RT test
```

Comment:
Please add that this is now the v17 test, similar to your debug test. I would also suggest adding the resolution, as is done in control_p8 for example ``at at C96L127``. Here it would be C96L149?

Reply:
Yes, this is at C96L149.